python/conversion/pocolog.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
+
+# Pocolog bindings
+from pocolog_pybind import *
+
+import tqdm
+import warnings
+import numpy as np
+
+from data.format import Events
+
+class EventAccumulator:
+    def __init__(self):
+        self.x = list()
+        self.y = list()
+        self.p = list()
+        self.t = list()
+
+    def add_event(self, event):
+        self.x.append(event.x)
+        self.y.append(event.y)
+        self.p.append(int(event.polarity))
+        self.t.append(event.ts.to_microseconds())
```

Comment:
@magehrig Actually. This part of the code is different in EventAcumulator for ros and for pocolog format. Pocolog timestamp does not have to_nsec method and changing it is not in my hands.

I am open to other suggestions but it seems we need to have two separated EventAccumulator classes.

Reply:
You can create a Base class with all but the non-shareable functionality